aim/tests/unit/agent/aid_universes/test_converter.py

```diff
@@ -103,8 +103,8 @@ def _test_reverse_map(self, resource_type, expected):
         reverse = converter.reverse_resource_map[resource_type]
         self.assertEqual(len(expected), len(reverse),
                          '\nExpected:\n%s\nFound:\n%s' %
-                            (pprint.pformat(expected),
```

Comment:
there are a bunch of indentations modified in pre existing code, please bring them back to their previous state unless they are breaking the UTs

Reply:
Those are coming from the rebase(all were done to get past PEP8). I will do a diff with master and revert if any are added by me.